lib/matplotlib/cbook/__init__.py

```diff
@@ -344,14 +344,14 @@ def is_hashable(obj):
 
 
 def is_writable_file_like(obj):
-    """return true if *obj* looks like a file object with a *write* method"""
+    """Return whether *obj* looks like a file object with a *write* method."""
     return callable(getattr(obj, 'write', None))
 
 
 def file_requires_unicode(x):
     """
-    Returns `True` if the given writable file-like object requires Unicode
-    to be written to it.
+    Returns whether the given writable file-like object requires Unicode to be
+    written to it.
```

Comment:
Personally, I like the original wording a bit better because it states what is returned, rather than implying a boolean result via the word "whether", which might be confusing for non-english-native readers.

Reply:
A quick grep shows that there's a *lot* of places where we're using `Return whether` to indicate a boolean return type (though admittedly I may be responsible for quite a few of them, as I also prefer that wording).  It's also quite common in the CPython docs (for example).